plans/ocean-mining-character-system-phase1.md

```
@@ -0,0 +1,418 @@
+強角色到新台只能拿 10% 產出。
+
+**任務**
+- [ ] 新增 `services/familiarity_service.go`：依 `cumulative_watch_seconds` 算熟悉度
+  - 0→60min：10%→50% 線性；60→300min：50%→100% 線性
+  - 30 天未觀看後每天 -1%，下限 10%
+- [ ] heartbeat tick 累加 `cumulative_watch_seconds` 並更新 `last_watched_at`
+- [ ] 在 `points_service.go` T-Points 產出路徑套用 `familiarity`（XP 路徑不套）
+- [ ] 單元測試：曲線插值邊界、衰減、新台 10% 驗證
+
+**介面／規格**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)